DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/WebCrawler.java

```diff
@@ -58,6 +72,25 @@ public static double getWebsiteFactor(Entity keywordCampaignEntity, Entity DSACa
         return 1;
     }
 
+    /*
+     * Converts the SQR data from a hash map to an embedded entity (entity will be embedded in the associated DSA campaign entity).
+     * The SQR should have first been populated by calling getWebsiteFactor(); otherwise, the SQR will be empty.
+     */
+    public EmbeddedEntity getSQR() {
+        EmbeddedEntity SQREmbeddedEntity = new EmbeddedEntity();
+        SQREmbeddedEntity.setProperty("numLines", SQR.size());
+
+        int lineNum = 1;
```

Comment:
add a comment explaining why its 1 and not 0

Reply:
I just chose 1 because when you read tables you never start at the 0th line - I feel like that doesn't really require an explanation.